### ISSUE TITLE
wrong flood attributes

### ISSUE THREAD

**Issue by AidasK** [svgdotjs/svg.filter.js#35]
```
draw.text(function(add) {
        add.tspan('bla').filterWith(function(add) {
            add.flood('#ff0000', 0.5);
        });
    });
```
creates:
```
<defs><filter id="SvgjsFilter1001"><feFlood id="SvgjsFeFlood1000" result="SvgjsFeFlood1000" in="SourceGraphic" opacity="0.5" color="#ff0000"></feFlood></filter></defs>
```

it should be "flood-opacity" instead of opacity and "flood-color" instead of color.

as stated in https://www.w3.org/TR/SVG11/filters.html#feFloodElement

tested with 
    "@svgdotjs/svg.filter.js": "3.0.5",
    "@svgdotjs/svg.js": "3.0.12",

**Comment by Fuzzyma** [comment]
Nice catch! Can you create a PR to fix this?

**Comment by AidasK** [comment]
done https://github.com/svgdotjs/svg.filter.js/pull/36, tested localy

**Comment by Fuzzyma** [comment]
merged and released as 3.0.6
